openssl/src/bn/mod.rs

```diff
@@ -473,6 +473,11 @@ impl BigNum {
         n
     }
 
+    pub unsafe fn into_raw(self) -> *mut ffi::BIGNUM {
```

Comment:
the ptr::replace on 478 is unsafe, so I need to mark this unsafe. I could place use an unsafe{} block, but I don't think that will be valuable since this is returning a raw pointer anyway.